wps-cloudformation-template.yaml

```diff
@@ -57,7 +57,44 @@ Parameters:
   geoserverWpsEndpoint:
     Type: String
     Default: 'http://geoserver-wps.aodn.org.au/geoserver/wps'
+  environment:
+    Type: String
+    Default: dev
+    AllowedValues:
+      - production
+      - systest
+      - sandbox
+      - dev
+  createProdSystestBucket:
+    Type: String
+    Default: false
+    AllowedValues:
+      - true
+      - false
+Mappings:
+  EnvironmentMap:
+    production:
+      BucketName: "aws-wps-production"
+    systest:
+      BucketName: "aws-wps-systest"
+Conditions:
+  CreateProdSystestBucket: !And
```

Comment:
Doesn't need to mention environment names, could just be something like "CreateRetainedBucket"

Reply:
Make sense.